packages/evm/src/evm.ts

```diff
@@ -234,18 +215,7 @@ export class EVM {
 
   protected readonly _emit: (topic: string, data: any) => Promise<void>
 
-  /**
-   * EVM async constructor. Creates engine instance and initializes it.
-   *
-   * @param opts EVM engine constructor options
-   */
-  static async create(opts: EVMOpts = {}): Promise<EVM> {
-    const evm = new this(opts)
-    await evm.init()
-    return evm
-  }
-
-  constructor(opts: EVMOpts) {
+  constructor(opts: EVMOpts = {}) {
```

Comment:
This is a bit of a side discussion, but since I am seeing this change/taking-over here:

Is there any difference in semantics between doing `opts: EVMOpts = {}` and `opts?: EVMOpts`?

I can vaguely remember a discussion around this, but absolutely not the outcome any more.

Atm we use this a bit mixed - especially for the `opts` part. Would be interesting to recall and see what benefits/drawbacks/differences are (if there are any).

Reply:
I think the difference is that if you use `opts?` it is undefined by default, while now it is `{}` by default. (I had to add this because some tests did not pass a param, and therefore if you read `opts.common` (opts is thus `undefined` if you do not default it to `{}`) it throws.

Reply:
Hmm, yeah, makes sense. Maybe we want to unify this before the releases (to default to `{}` for `opts` also for other libraries).